aws/resource_aws_acmpca_certificate_authority_test.go

```diff
@@ -496,6 +520,85 @@ func testAccCheckAwsAcmpcaCertificateAuthorityExists(resourceName string, certif
 	}
 }
 
+func testAccCheckAwsAcmpcaCertificateAuthorityActivateCA(certificateAuthority *acmpca.CertificateAuthority) resource.TestCheckFunc {
```

Comment:
This function activates a `ROOT` CA following the process described in the [documentation](https://docs.aws.amazon.com/acm-pca/latest/userguide/PCACertInstall.html#InstallRoot). It can be used in other tests where an `Active` CA is required.